autobuild/custom.yaml

```diff
@@ -16,14 +16,16 @@
     # DO NOT DELETE SNODE ENTRY
     - name: SNODE # DO NOT DELETE
       image: blocknetdx/servicenode:latest # DO NOT DELETE
-      config_mount_dir: /snode # DO NOT DELETE
-      data_mount_dir: /snode # DO NOT DELETE
-    - name: BTC # Requires 415.5 GB of disk (7 June, 2021)
+      config_mount_dir: /exr/data/snode # DO NOT DELETE
+      data_mount_dir: /exr/data/snode # DO NOT DELETE
+    - name: AVAX 
+      image: avaplatform/avalanchego:v1.6.1
+      data_mount_dir: /exr/data/snode
```

Comment:
The AVAX entry should be deleted from `custom.yaml`, IMHO. `custom.yaml` should just be a very bare bones example of the most basic services that every SNode will definitely deploy. AVAX will *not* be deployed by *all* Nodes.

Reply:
`custom.yaml` file is the default file if none is passed to `app.py`. we can delete it and its occurrence in `app.py`.

Reply:
> `custom.yaml` file is the default file if none is passed to `app.py`. we can delete it and its occurrence in `app.py`.

I think it's a good idea to delete `custom.yaml` from the repo, but I think we should leave its occurrence as the default `.yaml` used by `app.py` for now, at least until your fancy new whiz-bang user-friendly `.yaml` writing stuff is ready.